aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -0,0 +1,315 @@
+#include <ATen/native/quantized/affine_quantizer.h>
+
+#ifdef USE_FBGEMM
+#include <fbgemm/QuantUtils.h>
+#endif
+#ifdef __ARM_NEON__
+#include <arm_neon.h>
+#endif
+
+namespace at {
+namespace native {
+
+DEFINE_DISPATCH(quantize_tensor_affine_stub);
+DEFINE_DISPATCH(quantize_tensor_per_channel_affine_stub);
+DEFINE_DISPATCH(dequantize_tensor_affine_stub);
+DEFINE_DISPATCH(dequantize_tensor_per_channel_affine_stub);
+
+namespace {
+
+void checkCPUTensor(std::string fn_name, Tensor t) {
```

Comment:
please change `std::string` to `const std::string&` as well, same for the other functions, thanks

Reply:
Sure, fixed